### ISSUE TITLE
Issue where dashboard/my-quizzes is not showing the first time it's clicked and then showing every subsequent time. It is also not clearing for next user.

### ISSUE THREAD

**Issue by AndrewRLloyd88** [andreiskandar/quiz-app#42]

**Comment by AndrewRLloyd88** [comment]
When clicking Browse after a user has logged in there is nothing displaying the first time, after you return to the page you are seeing the previous AJAX request showing. It also seems to lag behind when a new user logs in.

**Comment by AndrewRLloyd88** [comment]
![quiz selection](https://user-images.githubusercontent.com/64162562/93838428-7471ac00-fc3e-11ea-89c5-1ec5effcaf3e.png)

**Comment by AndrewRLloyd88** [comment]
![BottomRequest](https://user-images.githubusercontent.com/64162562/93838466-9d923c80-fc3e-11ea-8d51-841b24aebdb8.png)

The bottom request indicates we are successfully making a GET to our DB but it's not showing on screen. I feel this is to do with the views_manager

**Comment by AndrewRLloyd88** [comment]
Fixed with help from Jensen simplified views manager to work differently now:

 window.views_manager.show("browsePublicQuizzes", quizDomElem); at the end of browse-public-quizzes

$(() => {
  const $main = $("#main-content");
  window.views_manager = {};
  window.views_manager.show = (item, domElem) => {
    // $login.detach();
    // $questionForm.detach();
    // $quizForm.detach();
    // $dashboard.detach();
    // $quizDomElem.detach();

    $main.empty();
    $main.append(domElem);

Now working principle in views_manager
